src/builtindict.js

```diff
@@ -110,4 +107,17 @@ Sk.builtins = {
     "coerce"    : Sk.builtin.coerce,
     "intern"    : Sk.builtin.intern
 };
-Sk.exportSymbol("Sk.builtins", Sk.builtins);
+
+Sk.setupObjects = function (py3) {
+    if (py3) {
```

Comment:
should we also remove `reduce` in python three? it's moved to functools.

Reply:
We can do that, but shouldn't someone implement ```functools``` first and remove it from builtins as a part of that?  It seems like a better story if someone asks "where did reduce go?" to be able to answer "it moved to functools, where it belongs in Python 3", rather than "it's gone".

Reply:
I agree I thought we had some functools already 😄